SofaKernel/modules/SofaBaseCollision/src/SofaBaseCollision/DefaultContactManager.cpp

```diff
@@ -153,6 +153,16 @@ void DefaultContactManager::createNewContacts(const core::collision::ContactMana
 
             std::string responseUsed = getContactResponse(model1, model2);
 
+            // To be removed at v22.06
+            std::map<std::string,std::string>::iterator it;
+            for(it=renamingResponseMethod.begin(); it!=renamingResponseMethod.end(); ++it)
+            {
+               if(responseUsed == it->first)
+               {
+                   msg_warning() << "Options for data \"response\" changed since #2522: please use "<< it->second << " instead of " << it->first;
+               }
+            }
```

Comment:
```suggestion
            const auto it = renamingResponseMethod.find(responseUsed);
            msg_warning_when(it != renamingResponseMethod.end()) << "Options for data \"response\" changed since #2522: please use "<< it->second << " instead of " << it->first;
```

Reply:
Also, in the case `responseUsed` corresponds to the old name, would you like to set `responseUsed` to the new name automatically?